doc/how-to/contribute-custom-widget.adoc

```diff
@@ -59,6 +59,12 @@ input EditSliderInput {
 union EditSliderPayload = SuccessPayload | ErrorPayload
 ----
 
+IMPORTANT: Make sure the type of the widget declared in the GraphQL Schema (`type Slider` above) matches the name of the Java class implementing the widget (`Slider.java`).
+
+IMPORTANT: When adding new widgets, note that if you use the same field name as another existing widget it must have the exact same type.
+For example, several of the core widgets (`Textfield`, `LabelWidget` among others) define a field `value: String!`.
+You can not name one of your custom widget's field `value` unless it is also typed `String!`.
```

Comment:
Not completely true; it's possible to reuse the same name, for fields with different types, but then the query must use unique aliases.
See for example in `FormEventFragment.ts`:

```
  fragment textfieldFields on Textfield {
    label
    iconURL
    stringValue: value
```